Northstar.Client/mod/resource/northstar_client_localisation_italian.txt

```diff
@@ -156,12 +156,12 @@ Premi Sì se sei d'accordo. Questa scelta può essere modificata in qualsiasi mo
 		"INFECTION_YOU_ARE_LAST_SURVIVOR" "Sei l'Ultimo Sopravvissuto!"
 		"INFECTION_SURVIVE_LAST_SURVIVOR" "Sopravvivi."
 
-		"PL_tffa", "Titan Free for All"
-		"PL_tffa_lobby", "Titan Free for All Lobby"
+		"PL_tffa", "Titan Tutti contro Tutti"
+		"PL_tffa_lobby", "Titan Tutti contro Tutti Lobby"
```

Comment:
line 160 can be changed in the same forms as the other "lobby" lines, changing it to "Lobby: Titan Tutti contro Tutti"

Reply:
Done in c7bfeb753b758b9366a90f40e7a66aaca6750f84